sections/04_study.md

```diff
@@ -709,101 +709,92 @@ the next few years.
-Recently, multi-layer, recurrent networks (and convolutional
-networks) have been applied to microbiome genotype-phenotype, with Ditzler et
-al. being the first to associate soil samples with pH level using multi-layer
-perceptrons, deep-belief networks, and recursive neural networks (RNNs)
-[@tag:Ditzler3]. Besides classifying the samples appropriately, Ditzler shows
-that internal phylogenetic tree nodes inferred by the networks are
-appropriate features representing low/high pH, which can provide additional
-useful information and new features for future metagenomic sample comparison.
- Also, an initial study has show promise of these networks for diagnosing
-disease [@tag:Faruqi].  
-
-There are still a lot of challenges with applying deep neural networks to metagenomics problems.  They are not ideal for microbial/functional composition->phenotype classification because most studies contain tens of samples (~20->40) and
-hundreds/thousands of features (aka species).  Such underdetermined/ill-conditioned problems
-are still a challenge for deep neural networks that require many more training examples than
-features to sufficiently converge the weights on the hidden layers.  Also, due to convergence issues
-(slowness and instability due to large neural networks modeling very large datasets [@arxiv:1212.0901v2]),
-taxonomic classification of reads from whole genome sequencing seems out of reach at the moment for deep neural
-networks -- due to only thousands of full-sequenced genomes as compared to hundreds of thousands of 16S rRNA sequences
-available for training.
-
-However, because recurrent neural networks are showing success for base-calling (and thus removing the large error in
-the measurement of a pore's current signal) for the relatively new Oxford Nanopore sequencer [@tag:Boza], there is hope that
-the process of denoising->organism/function classification can be combined into one step in using powerful LSTM's. LSTM's
-are working miracles in raw speech signal->meaning translation [@tag:Sutskever], and combining steps in metagenomics are not
-out of the question.  For example, metagenomic assembly usually requires binning then assembly, but could deep neural nets
-accomplish both tasks in one network? Does functional/taxonomic classification need to be separate processes?  The largest
-potential in deep learning is to learn "everything" in one complex network, with a plethora of labeled (reference) data and
-unlabeled (microbiome experiments) examples.
+Metagenomics, which refers to the study of genetic material, 16S rRNA and/or
+whole-genome shotgun DNA from microbial communities, has revolutionized the
+study of micro-scale ecosystems within us and around us. In recent years,
+machine learning has proved to be a powerful tool for metagenomic analysis. 16S
+rRNA has long been used to deconvolve mixtures of microbial genomes, yet this
+ignores >99% of all reads. Subsequent tools aimed to classify 300-3000bp reads
```

Comment:
Well.. if you're just sequencing 16S rRNA, then all reads are from 16S rRNA.  So, it's not "ignores >99% of all reads" but ">99% of the genome content".